images.CI/linux-and-win/InstallAzCopy.ps1

```diff
@@ -0,0 +1,28 @@
+Write-Host 'Acquiring AzCopy executable...'
```

Comment:
lets not install azcopy in runtime and pre-install it onto the agents, I think @Alexey-Ayupov can help with this

Reply:
We can get rid of azcopy and use `az storage blob copy start` command if we would like. (https://learn.microsoft.com/en-us/azure/virtual-machines/scripts/copy-managed-disks-vhd-to-storage-account)

Reply:
Replaced `azcopy` with `az storage blob copy start`